plugins/infinispan90/src/main/java/org/radargun/service/JGroups4Dumper.java

```diff
@@ -0,0 +1,58 @@
+package org.radargun.service;
+
+import java.lang.reflect.Field;
+import java.util.Map;
+import java.util.concurrent.ThreadPoolExecutor;
+import org.jgroups.protocols.TP;
+import org.jgroups.stack.Protocol;
+import org.jgroups.stack.ProtocolStack;
+import org.jgroups.util.TimeScheduler;
+import org.radargun.utils.Utils;
+
+/**
+ * JGroupsDumper modified for JGroups 4
+ *
+ * @author Roman Macor (rmacor@redhat.com)
+ */
+public class JGroups4Dumper extends JGroupsDumper {
+
+   public JGroups4Dumper(ProtocolStack protocols, long interval) {
+      super(protocols, interval);
+   }
+
+   @Override
+   protected void addDumpers(){
+      dumpers.add(new UNICAST3Dumper());
+      dumpers.add(new JGroups4Dumper.TPDumper());
+   }
+
+   @Override
+   protected Map<String, Object> getStats(Protocol protocol){
+      return stack.dumpStats();
+   }
+
+   protected static class TPDumper extends JGroupsDumper.TPDumper {
+      @Override
+      public void dump(Protocol protocol) {
+         TP tp = (TP) protocol;
+         boolean dump = false;
+         dump = checkThreadPool((ThreadPoolExecutor) tp.getThreadPool(), "Regular") || dump;
```

Comment:
I'm not sure what are you referring to, what do you want me to rename? Anyway names JGroups and Common seem to general to me.